shodan/cli/alert.py

```diff
@@ -132,22 +134,30 @@ def alert_domain(domain, triggers):
 
         if domain_parse.subdomain:
             domain_ips = set([record['value'] for record in info['data']
-                              if record['subdomain'] == domain_parse.subdomain])
+                              if record['subdomain'] == domain_parse.subdomain and
+                              not ip_address(record['value']).is_private])
         else:
-            domain_ips = set([record['value'] for record in info['data']])
+            domain_ips = set([record['value'] for record in info['data']
+                              if not ip_address(record['value']).is_private])
 
-        # Create the actual alert
-        click.secho('Creating alert...', dim=True)
-        alert = api.create_alert('__domain: {}'.format(domain), list(domain_ips))
+        if not domain_ips:
+            flag = False
+            click.secho('No external IPs were found to be associated with this domain. '
```

Comment:
Why not just `raise click.ClickException('No external ...')`? Then we don't need to use a `flag` variable to track the return status and I think it's technically correct to error out if it wasn't possible to generate an asset group due to lacking IPs.

Reply:
ahhh yep! I'll make that change!

Reply:
@achillean Makes sense! Changes made!